cli/src/config/templates.toml

```diff
@@ -42,13 +42,25 @@ label("tag", name) ++ format_ref_targets(self) ++ "\n"
 '''
 
 # TODO: "op_log" is not a great label name for op_summary. We could replace it
-# with `operation` and insert this new label throughout the op log templates.
+# with `operation` in `colors.toml`, and insert "operation summary" labels in
+# the caller. https://github.com/martinvonz/jj/pull/4602#discussion_r1791323154
+# TODO: Here and elsewhere, the end time for operations is more meaningful than
+# the start time. https://github.com/martinvonz/jj/pull/4602#discussion_r1791207491
 op_summary = '''
 label(
   "op_log",
-  if(root,
-    separate(" ", self.id().short(), label("root", "root()")),
-    separate(" ", self.id().short(), format_time_range(self.time()), self.description().first_line())
+  concat(
+    self.id().short(),
+    if(!root, concat(
+       " (",
+       format_timestamp(self.time().start()),
+       ")"
+    )),
```

Comment:
nit: perhaps, we can use `separate()` and/or `surround()` to deal with whitespace
```
separate(" ",
  self.id().short(),
  if(root, .., ..),
  self.description().first_line(),
)
```

Reply:
Ah, forgot that `surround` existed, great!